composer.json

```diff
@@ -83,7 +83,7 @@
         "post-update-cmd": [
             "@auto-scripts"
         ],
-        "tests": "phpunit",
+        "tests": "phpunit --cache-result-file=/tmp/.phpunit.result.cache",
```

Comment:
Tihle je zajimavy.. `composer ci`  v notification service se pousti v `app` targetu, ktery bezi za non root usera, takze nema pravo zapisu do slozky projektu (kam se defaultne dava ta cache), ale presto to prochazi :thinking: any idea?

Reply:
ano, prudí to tam taky :) 

![image](https://user-images.githubusercontent.com/4319320/129420163-1866bac9-77cc-4213-a045-cb1ba44f8748.png)

on je to jen warning, ale štval mě tam

Reply:
:bulb: aha, to jsem si nevsiml.. nicmene pro CI by se ta cache klidne mohla uplne vypnout. Bohuzel koukam, ze PHPUnit ma v podstate nulovy moznosti konfigurace pres ENV :( takze asi takle zatim no :+1:

Reply:
Jo myslím, že to bude ještě takovej další běh to vyladit na těch "nových" imagích.